client/state/themes/themes/reducer.js

```diff
@@ -7,35 +7,35 @@ import transform from 'lodash/object/transform';
 /**
  * Internal dependencies
  */
-import ThemeConstants from 'lib/themes/constants';
+import ActionTypes from '../action-types';
 
 const initialState = fromJS( {
 	themes: {},
 	currentSiteId: 0
 } );
 
-function add( newThemes, themes ) {
-	return themes.merge( transform( newThemes, ( result, theme ) => {
+function add( newThemes, ts ) {
```

Comment:
Sry, in a previous iteration `themes`would've shadowed a function name. Renamed back to `themes` in 3f93bebf328a4c51d16cea380d6d89dc55c77561.